src/controller/group-controller.js

```diff
@@ -0,0 +1,134 @@
+
+import { PrismaClient } from "@prisma/client";
+// import prisma from 'prisma'
+import groupRepository from "../repository/group-repository.js";
+import tagRepository from "../repository/group-tag-repository.js";
+import groupService from "../service/group-service.js";
+
+const prisma = new PrismaClient();
```

Comment:
```suggestion
import groupRepository from "../repository/group-repository.js";
import tagRepository from "../repository/group-tag-repository.js";
import groupService from "../service/group-service.js";
```
PrismaClient 모듈과 인스턴스는 없어도 될 것 같아요